hudi-common/src/test/java/org/apache/hudi/common/testutils/HoodieTestUtils.java

```diff
@@ -279,6 +287,23 @@ public static String createDataFile(String basePath, String partitionPath, Strin
     return fileID;
   }
 
+  public static void createMarkerFile(String basePath, String partitionPath, String instantTime, String dataFileName) throws IOException {
```

Comment:
we should keep these to just HoodieClientTestUtils. since markers are just an artifact of the client